frontend/src/core/comments/components/AddComment.js

```diff
@@ -25,69 +34,91 @@ export default function AddComments(props: Props) {
-            commentInput.current.rows = currentRows;
-        }
-        else {
-            commentInput.current.rows = maxRows;
+    function keyBindingFn(e: SyntheticKeyboardEvent<>) {
+        if (e.keyCode === 13 && e.shiftKey === false) {
+            return 'submit-on-enter'; 
         }
+        
+        return getDefaultKeyBinding(e)
     }
 
-    const handleOnKeyDown = (event: SyntheticKeyboardEvent<>) => {
-        if (event.keyCode === 13 && event.shiftKey === false) {
-            submitComment(event);
+    const handleKeyCommand = (command: string) => {
+        if (command === 'submit-on-enter') {
+            submitComment();
+            return 'handled';
         }
+        return 'not-handled';
     }
 
-    const submitComment = (event: SyntheticEvent<>) => {
-        event.preventDefault();
-        const comment = commentInput.current.value;
+    const clearEditorContent = () => {
+        let contentState = editorState.getCurrentContent();
+        const firstBlock = contentState.getFirstBlock();
+        const lastBlock = contentState.getLastBlock();
+        const allSelected = new SelectionState({
+            anchorKey: firstBlock.getKey(),
+            anchorOffset: 0,
+            focusKey: lastBlock.getKey(),
+            focusOffset: lastBlock.getLength(),
+            hasFocus: true
+        });
+        contentState = Modifier.removeRange(contentState, allSelected, 'backward');
+        editorState = EditorState.push(editorState, contentState, 'remove-range');
+        editorState = EditorState.forceSelection(editorState, contentState.getSelectionAfter());
+        setEditorState(editorState);
+    }
 
-        if (!comment) {
+    const submitComment = () => {
+        if (!editorState.getCurrentContent().hasText()) {
             return null;
         }
-
+        
+        const rawContentState = convertToRaw(editorState.getCurrentContent());
+        const comment = draftToHtml(
+            rawContentState,
+        )
+        
         addComment(comment, translation);
 
-        commentInput.current.value = '';
-        commentInput.current.rows = minRows;
-    };
+        clearEditorContent();
+    }
 
     return <div className='comment add-comment'>
         <UserAvatar
             username={ username }
             imageUrl={ imageURL }
         />
-        <form className='container'>
-            <Localized
-                id='comments-AddComment--input'
-                attrs={{ placeholder: true }}
-            >
-                <textarea
-                    autoFocus
-                    name='comment'
-                    dir='auto'
-                    placeholder={ `Write a comment…` }
-                    rows={ minRows }
-                    ref={ commentInput }
-                    onChange={ handleOnChange }
-                    onKeyDown={ handleOnKeyDown }
+        <div className='container'>
+            <div className='editor-wrapper' onClick={ focusEditor }>
```

Comment:
If we're focusing on click on the wrapper, we should also set `cursor: text` on it.